conference.js

```diff
@@ -534,7 +534,7 @@ export default {
      * @returns {Promise<JitsiLocalTrack[]>, Object}
      */
     createInitialLocalTracks(options = {}) {
-        const errors = {};
+        let errors = {};
```

Comment:
Do you need to modify errors? I see only assigning properties. Sorry if I'm blind!

Reply:
I've just saw it. I'm blind :) discard this one!